charts/fluent-bit.yaml

```diff
@@ -11,7 +11,7 @@ podAnnotations:
 
 resources:
   limits:
-    memory: 100Mi
+    memory: 300Mi
```

Comment:
:thinking: why are we limiting its memory in the first place? IIRC, `fluentbit` has the tendency to generate memory usage spikes, do we really want it to get OOMKilled if it were to happen? We've seen cases where we had to remove this limit to let the spike pass (was reading too many things on start, not sure why).

Reply:
I don't know for the initial reason TBH, I will try to find the reason

Reply:
Ok just checked and it's just because it was set by default in the old helm chart value file, but with the new chart it's no longer set by default so I agree to remove it